ERCS/erc-7604.md

```diff
@@ -0,0 +1,190 @@
+---
+eip: 7604
+title: ERC-1155 Permit Approvals
+description: Permit approvals for ERC-1155 tokens
+author: calvbore (@calvbore), emiliolanzalaco (@emiliolanzalaco)
+discussions-to: https://ethereum-magicians.org/t/proposal-for-a-new-eip-erc-2612-style-permits-for-erc1155-nfts/15504
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-01-27
+requires: 165, 712, 1155, 5216
```

Comment:
```suggestion
requires: 165, 712, 1155, 1271, 2098, 5216
```

You need to read both 1271 and 2098 to implement the signature validation, so they're required reading.